src/backend/base/langflow/components/embeddings/HuggingFaceInferenceAPIEmbeddings.py

```diff
@@ -1,32 +1,59 @@
 from langchain_community.embeddings.huggingface import HuggingFaceInferenceAPIEmbeddings
 from pydantic.v1.types import SecretStr
+import requests
+from urllib.parse import urlparse
 
 from langflow.base.models.model import LCModelComponent
 from langflow.field_typing import Embeddings
 from langflow.io import MessageTextInput, Output, SecretStrInput
 
 
 class HuggingFaceInferenceAPIEmbeddingsComponent(LCModelComponent):
-    display_name = "HuggingFace Embeddings"
-    description = "Generate embeddings using Hugging Face Inference API models."
-    documentation = "https://github.com/huggingface/text-embeddings-inference"
+    display_name = "HuggingFace Embeddings Inference"
+    description = "Generate embeddings using HuggingFace Text Embeddings Inference (TEI)"
+    documentation = "https://huggingface.co/docs/text-embeddings-inference/en/index"
```

Comment:
Can change this to `https://huggingface.co/docs/text-embeddings-inference/index` - the localization is not needed.